docs/reference/action_list.rst

```diff
@@ -98,7 +98,12 @@ Here is an example::
             ->add('_action', null, [
                 'actions' => [
                     'show' => [],
-                    'edit' => [],
+                    'edit' => [
+                        // You may add custom link parameters used to generate the action url
+                        'link_parameters' => [
```

Comment:
Good decision to rename ambiguous "parameters" for a more consistent "link_parameters" :+1:
Could we also update the table at "Available types and associated options"? It has a entry for "actions" type that could contain an explanation similar to "url" type.

Reply:
I'm not sure how to correctly do this.
`url` has an option `route` which is an array with `name` and `parameters`.

```
url => [ route => [ name => ..., parameter => [] ] ]
```

But in my case, `actions` has an option `actions`, and each `action` has an option `link_parameters`. 
```
actions => [ actions => [ edit => [ link_parameters => [] ] ] ]
```
If I use a similar exaplanation, user could believe it's
```
actions => [
   actions => [ edit => []],
   link_parameters => []
]

Reply:
I was thinking in something like this:

```
+---------+-------------------+- --------------------------------+
| actions | actions           | List of available actions        |
+         +-------------------+- --------------------------------+
|         |   edit            | Give a route to generate the url |
+         +                   +                                  +
|         |   link_parameters | Route parameters                 |
+---------+-------------------+----------------------------------+
```

Reply:
I made something like this @phansys